core/postfix/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM alpine:3.8
 
 RUN apk add --no-cache postfix postfix-pcre rsyslog \
  && python3 py3-pip \
- && pip3 install --upgrade pip \
+    pip3 install --upgrade pip \
```

Comment:
@kaiyou  Wrong line, the build fails on line 3 /4. `apk add` needs to install `python3 and `py3-pip`. With the `&&` in front, you just try to run `python3` which is not installed.